s3/public.go

```diff
@@ -384,3 +384,62 @@ func (storage *PublishedStorage) RenameFile(oldName, newName string) error {
 
 	return storage.Remove(oldName)
 }
+
+// SymLink creates a copy of src file and adds link information as meta data
+func (storage *PublishedStorage) SymLink(src string, dst string) error {
+
+	params := &s3.CopyObjectInput{
+		Bucket:     aws.String(storage.bucket),
+		CopySource: aws.String(src),
```

Comment:
I think we need to do `filepath.Join(storage.prefix, src)` here as well

Reply:
Changed.